From d6fa42450143995291d417ca22e32db5b73b32e7 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 24 Jul 2012 09:56:28 +0800
Subject: [PATCH 1798/2508] ENGR00218067 mx6sl LDO_BYPASS: enable LDO BYPASS
 in mx6sl by default

To validate LDO bypass function fully, enable CONFIG_MX6_INTER_LDO_BYPASS
on u-boot and kernel, only for mx6sl.
Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/configs/imx6s_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6s_defconfig b/arch/arm/configs/imx6s_defconfig
index 231b4f1..78efc6b 100644
--- a/arch/arm/configs/imx6s_defconfig
+++ b/arch/arm/configs/imx6s_defconfig
@@ -319,7 +319,7 @@ CONFIG_MACH_MX6Q_SABRESD=y
 # MX6 Options:
 #
 # CONFIG_IMX_PCIE is not set
-# CONFIG_MX6_INTER_LDO_BYPASS is not set
+CONFIG_MX6_INTER_LDO_BYPASS=y
 CONFIG_ISP1504_MXC=y
 # CONFIG_MXC_IRQ_PRIOR is not set
 CONFIG_MXC_PWM=y
-- 
1.7.7.4

